package.json

```diff
@@ -61,22 +64,20 @@
     "extract-text-webpack-plugin": "^0.8.2",
     "file-loader": "^0.8.4",
     "filesaver.js": "^0.2.0",
-    "jest-cli": "^0.5.1",
+    "jest-cli": "^22.0.6",
```

Comment:
There are two factors contributing to failed travis tests:

1.  [Jest 22 has stopped supporting Node 4](https://github.com/facebook/jest/blob/master/CHANGELOG.md#chore--maintenance-4). 

2. Two of our plugins for React, `react-loader` and `react-bootstrap`, don't yet support react 16.  `React-loader` has an [available and approved but unmerged branch](https://github.com/CognizantStudio/react-loader/pull/66) that supports react 16, but only 16.0. `React-bootstrap` doesn't have one that I could find; it depends on` react-overlays`, which has actual conflicts with react 16. Node 4 treats plugin dependencies as an error; Node 5 and 6 as a warning, so this prevents Node 4 from building properly.

We're not using any `react-overlays`, so the _actual_ incompatibilities between `react-bootstrap` and react 16 don't affect us. All tests can be passed on one's workstation by switching to` jest-cli` version 21, building `climate-explorer-frontend` with npm 5 or 6, ignoring the warnings about peer dependencies, and then switching back to npm 4 to run the tests. This doesn't seem like a good long-term strategy for Travis and integration testing, however. Perhaps we should consider dropping support for Node 4 as well, and make an issue to update react-bootstrap and react-loader to react 16 friendly versions as soon as they're available.

Reply:
Fixed